Jobs/GenerateTag.yml

```diff
@@ -0,0 +1,69 @@
+## @file
+# Template file used to generate tags on ADO. This template requires that the
+# consumer specifies this repository as a resource named mu_devops.
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+  - name: base_version
+    displayName: The base version.
+    type: string
+    default: ""
+  - name: git_name
+    displayName: Name to use for creating tag.
+    type: string
+    default: ""
+  - name: git_email
+    displayName: Email to use for creating tag.
+    type: string
+    default: ""
+  - name: notes_file
+    displayName: Path to the notes file to generate.
+    type: string
+    default: "ReleaseNotes.md"
+
+jobs:
+  - job: Create_Release_Tag
+    steps:
+      - checkout: self
+        clean: true
+        fetchTags: true
+        persistCredentials: true

```

Comment:
Just curious, what is this for?

Reply:
This is required in order to push the generated tag and notes commit.